eks/install_eks.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+ 
+set +e
+
+## Install eksctl
+curl --silent --location \
+"https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_$(uname -s)_amd64.tar.gz" \
+| tar xz -C /tmp
+
+sudo mv /tmp/eksctl /usr/local/bin
+eksctl version
+
+## Configure AWS credentials 
+aws configure set aws_access_key_id "$ACCESS_KEY"
+aws configure set aws_secret_access_key "$SECRET_KEY"
+aws configure set default.region "$REGION"
+
+## Creating AWS cluster with two nodes and noty type t2.xlarge
+## in region us-east-2
+eksctl create cluster --name=kubera-cluster --node-type=t2.xlarge
```

Comment:
Can we make use of declarative way i.e. yaml to install EKS of desired nodes, ram, cpu, etc?